django-backend/fecfiler/devops/utils/load_test_utils.py

```diff
@@ -24,22 +44,30 @@ def create_load_test_committee_and_data(
         self.locust_data_generator = LocustDataGenerator(committee)
 
         logger.info(f"Creating {number_of_reports} reports")
-        reports = self.create_reports(number_of_reports)
+        reports = self.locust_data_generator.generate_form_3x(number_of_reports)
 
         logger.info(f"Creating {number_of_contacts} contacts")
-        contacts = self.create_contacts(number_of_contacts)
+        contacts = self.locust_data_generator.generate_contacts(number_of_contacts)
 
         single_transactions_needed = math.ceil(
             number_of_transactions * single_to_triple_transaction_ratio
         )
         logger.info(f"Creating {single_transactions_needed} single transactions")
-        self.create_single_transactions(single_transactions_needed, reports, contacts)
+        self.locust_data_generator.generate_single_schedule_a_transactions(
+            number_of_transactions,
```

Comment:
I think you want this to be `single_transactions_needed`

Reply:
@toddlees fixed, thanks!